language.txt

```diff
@@ -76,6 +76,32 @@ Unicode code point."
 `AbCdEf`
 
 
+# Bit string literals.
+
+# A backtick string beginning with 'b' is a bit string literal. 0 or 1
+# characters denote bits in a bit string. | characters are interpreted as below.
+# No other characters may appear. The emit the contents of the bit string's DER
+# encoding as a BIT STRING. (Big-endian bit order, prefixed with the number of
+# trailing padding bits)
+
+# This encodes as `0055`.
+b`10101010`
```

Comment:
You know what I'm going to say... :-)

(s/5/a/ in all these examples, or use 0101... instead of 1010...)

I will say, the bit order is annoying to write in code, but it's actually very convenient as a human, because we write everything big-endian. Leading byte aside, <code>b\`10101010\`</code> is just the byte `0b10101010`. Group them by fours to get hex, and `0b1010 = 0xa`.